src/App.js

```diff
@@ -1,15 +1,107 @@
 import React, { Component } from 'react';
+import ContactList from './components/contact-list.js';
+import SearchBar from './components/search-bar.js';
+import Button from './components/button.js';
+import axios from 'axios';
+
+const data = {
+  contacts: [
+    {
+      _id: 1,
+      name: '',
+      occupation: '',
+      avatar: ''
+    },
+  ]
+};
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data,
+      query: '',
+      selected: ['No Contacts Selected']
+    };
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:4000/contacts')
+      .then(resp => {
+        data.contacts = resp.data;
```

Comment:
Mutating the data object is dangerous, since in your implementation all instances of the Component share the same variable. Also see comment above. Just use state for that and remove the local variable above.